test/pantry-test.js

```diff
@@ -1,53 +1,68 @@
 import { expect } from 'chai';
+import User from '../src/user';
 import userData from '../src/data/users-data';
 import Recipe from '../src/recipe';
 import recipeData from '../src/data/recipe-data';
 import Pantry from '../src/pantry';
 
 
 describe('Pantry', () => {
-  let pantry, userInfo, recipe 
-  
-  beforeEach(() => {
-    userInfo = userData[0];
-    pantry = new Pantry(userInfo.pantry, userInfo.id);
-    recipe = new Recipe(recipeData[0]);
-  });
 
-  afterEach(() => {
-    userInfo = userData[0];
-  });
+  describe('instantiate', () => {
+    let pantry, userInfo
+  
+    beforeEach(() => {
+      userInfo = new User(userData[0]);
+      pantry = userInfo.pantry;
+    });
 
-  describe.only('instantiate', () => {
     it('should be an instance of Pantry class', () => {
       expect(pantry).to.be.an.instanceof(Pantry);
     });
+
   });
 
-  describe.only('method', () => {
+  describe('method', () => {
+    let pantry, userInfo, recipe 
+  
+    beforeEach(() => {
+      userInfo = new User(userData[0]);
+      pantry = userInfo.pantry;
+      recipe = new Recipe(recipeData[0]);
+    });
+
+    it('should consolidate duplicate ingredients', () => {
+      const ingredientKey = Object.keys(pantry.pantry)[0];
+      const ingredient1 = pantry.pantry[ingredientKey];
+
+      expect(Object.keys(pantry.pantry).length).to.equal(113)
```

Comment:
Maybe we can think about renaming 'pantry' at some point? Because a Pantry class with a pantry property can get a bit confusing, especially as we need to access it as an object with stacking dot/bracket notation.